### ISSUE TITLE
Laravel 5 Support

### ISSUE THREAD

**Issue by gravataLonga** [makzumi/laravel-calendar#28]
Any one can say me if this library work on laravel 5 ? and if isn't what is the plan for? 

Many Thanks.

**Comment by abernabe** [comment]
No, it doesn't work. Does anyone know an alternative that would work with minimum changes on the code?

**Comment by gravataLonga** [comment]
Appear no body want to work this on laravel 5, maybe @abernabe want join me on quest to transform this into laravel 5 support?

**Comment by skecskes** [comment]
@gravataLonga and @abernabe I made the required changes to be able to run it under Laravel 5 some time ago and asked @makzumi to merge it into his repository as branch laravel5. It was rejected 
https://github.com/makzumi/laravel-calendar/pull/31

Probably the best would be to create a new repo with laravel5 code.

**Comment by abernabe** [comment]
Why did they reject your changes @skecskes ?

At the moment I don't have time to work on this. Also, I guess that since that time, someone must have done an alternative library for calendars that works on Laravel 5.

**Comment by skecskes** [comment]
Hey @gravataLonga and @abernabe 

I don't remember what happened last time, but you are right. We can't wait longer, so I renamed namespaces and made necessary changes and my fork works now only under Laravel 5. I also added it to packagist so you can easily use it as `composer require skecskes/calendar` 
The repo is https://github.com/skecskes/laravel-calendar
